packages/cloud/reader/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "module",
```

Comment:
this seems like a nodejs 18 issue, it will read cwd package.json type as file type(cjs/esm), instread of the node_modules package.json type

Reply:
no, this is a expected behavior